web3-dev/smart-contracts/autonomous-smart-contract.rst

```diff
@@ -3,17 +3,17 @@
 Autonomous Smart Contract
 =========================
 
-In smart-contract world of Massa, you can create smart-contracts that can be called in the future.
+In smart-contract world of Massa, you can create smart-contracts that can be triggered in the future.
 
 This system works by sending messages in the future specifying the address and the function you want to trigger.
 
 You need to use the ABI `sendMessage(...)` in your smart contract if you want to send a message to wakeup a function in the future.
```

Comment:
I think this can be improved as well.

Reply:
I think we should use "emit async message = schedule a future execution" as a general vocabulary rule, instead of "waking up" and "sending".